OSX 12 tests are sometime timing out

E.g. in #1356 and #1349, although not in #1352.

Here's a dump of the tests section of the log from #1356:

```
2024-02-23T20:32:33.9496360Z ...../Users/runner/work/pyiron_base/pyiron_base/tests/archiving/test_export.py:49: FutureWarning: Calling float on a single element Series is deprecated and will raise a TypeError in the future. Use float(ser.iloc[0]) instead
2024-02-23T20:32:33.9542170Z   df_read["hamversion"] = float(df_read["hamversion"])
2024-02-23T20:32:33.9560290Z /Users/runner/work/pyiron_base/pyiron_base/tests/archiving/test_export.py:53: FutureWarning: Calling float on a single element Series is deprecated and will raise a TypeError in the future. Use float(ser.iloc[0]) instead
2024-02-23T20:32:33.9729270Z The job toy was saved and received the ID: 1
2024-02-23T20:32:33.9740370Z   df_exp["hamversion"] = float(df_exp["hamversion"])
2024-02-23T20:32:33.9802610Z .
2024-02-23T20:32:33.9919490Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:32:33.9951480Z 100%|██████████| 1/1 [00:00<00:00, 49.88it/s]
2024-02-23T20:32:33.9964460Z 
2024-02-23T20:32:34.0000380Z 0it [00:00, ?it/s]
2024-02-23T20:32:34.0013220Z 0it [00:00, ?it/s]
2024-02-23T20:32:34.0034450Z s
2024-02-23T20:32:34.0035640Z remove directory: /Users/runner/work/pyiron_base/pyiron_base/archive_folder/
2024-02-23T20:32:34.0037770Z 0it [00:00, ?it/s]
2024-02-23T20:32:34.0041990Z 0it [00:00, ?it/s]
2024-02-23T20:32:34.2560760Z 
2024-02-23T20:32:34.2561310Z The job toy was saved and received the ID: 1
2024-02-23T20:32:34.2562510Z 0it [00:00, ?it/s]
2024-02-23T20:32:34.2563060Z 0it [00:00, ?it/s]
2024-02-23T20:32:34.2639370Z /Users/runner/miniconda3/envs/my-env/lib/python3.12/tarfile.py:2221: DeprecationWarning: Python 3.14 will, by default, filter extracted tar archives and reject files or modify their metadata. Use the filter argument to control this behavior.
2024-02-23T20:32:34.2642270Z   warnings.warn(
2024-02-23T20:32:34.2857390Z 
2024-02-23T20:32:34.2974240Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:32:34.2975270Z 100%|██████████| 1/1 [00:00<00:00, 87.70it/s]
2024-02-23T20:32:34.3154260Z /Users/runner/miniconda3/envs/my-env/lib/python3.12/tarfile.py:2221: DeprecationWarning: Python 3.14 will, by default, filter extracted tar archives and reject files or modify their metadata. Use the filter argument to control this behavior.
2024-02-23T20:32:34.3157190Z   warnings.warn(
2024-02-23T20:32:34.4780130Z 
2024-02-23T20:32:34.4959390Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:32:34.4961310Z 100%|██████████| 1/1 [00:00<00:00, 56.58it/s]
2024-02-23T20:32:34.4994160Z .
2024-02-23T20:32:34.4994940Z 0it [00:00, ?it/s]
2024-02-23T20:32:34.4996220Z 0it [00:00, ?it/s]
2024-02-23T20:32:34.5039840Z /Users/runner/miniconda3/envs/my-env/lib/python3.12/tarfile.py:2221: DeprecationWarning: Python 3.14 will, by default, filter extracted tar archives and reject files or modify their metadata. Use the filter argument to control this behavior.
2024-02-23T20:32:34.5044770Z   warnings.warn(
2024-02-23T20:32:34.5202750Z 
2024-02-23T20:32:34.5367390Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:32:34.5368650Z 100%|██████████| 1/1 [00:00<00:00, 61.96it/s]
2024-02-23T20:32:34.5397760Z .
2024-02-23T20:32:34.5398450Z 0it [00:00, ?it/s]
2024-02-23T20:32:34.5399090Z 0it [00:00, ?it/s]
2024-02-23T20:32:34.5440000Z /Users/runner/miniconda3/envs/my-env/lib/python3.12/tarfile.py:2221: DeprecationWarning: Python 3.14 will, by default, filter extracted tar archives and reject files or modify their metadata. Use the filter argument to control this behavior.
2024-02-23T20:32:34.5442920Z   warnings.warn(
2024-02-23T20:32:34.5598640Z 
2024-02-23T20:32:34.5716770Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:32:34.5718200Z 100%|██████████| 1/1 [00:00<00:00, 86.24it/s]
2024-02-23T20:32:34.5749290Z .
2024-02-23T20:32:34.5749790Z 0it [00:00, ?it/s]
2024-02-23T20:32:34.5750480Z 0it [00:00, ?it/s]
2024-02-23T20:32:34.5797860Z /Users/runner/miniconda3/envs/my-env/lib/python3.12/tarfile.py:2221: DeprecationWarning: Python 3.14 will, by default, filter extracted tar archives and reject files or modify their metadata. Use the filter argument to control this behavior.
2024-02-23T20:32:34.5800640Z   warnings.warn(
2024-02-23T20:32:34.6042180Z /Users/runner/work/pyiron_base/pyiron_base/tests/archiving/test_import.py:58: FutureWarning: Calling float on a single element Series is deprecated and will raise a TypeError in the future. Use float(ser.iloc[0]) instead
2024-02-23T20:32:34.6047050Z   df_import["hamversion"] = float(df_import["hamversion"])
2024-02-23T20:32:34.6051860Z /Users/runner/work/pyiron_base/pyiron_base/tests/archiving/test_import.py:59: FutureWarning: Calling float on a single element Series is deprecated and will raise a TypeError in the future. Use float(ser.iloc[0]) instead
2024-02-23T20:32:34.6055720Z   df_original["hamversion"] = float(df_original["hamversion"])
2024-02-23T20:32:34.6103430Z 
2024-02-23T20:32:34.6219530Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:32:34.6220620Z 100%|██████████| 1/1 [00:00<00:00, 87.87it/s]
2024-02-23T20:32:34.6249850Z .
2024-02-23T20:32:34.6250260Z 0it [00:00, ?it/s]
2024-02-23T20:32:34.6250780Z 0it [00:00, ?it/s]
2024-02-23T20:32:34.6290730Z /Users/runner/miniconda3/envs/my-env/lib/python3.12/tarfile.py:2221: DeprecationWarning: Python 3.14 will, by default, filter extracted tar archives and reject files or modify their metadata. Use the filter argument to control this behavior.
2024-02-23T20:32:34.6293520Z   warnings.warn(
2024-02-23T20:32:34.6527780Z 
2024-02-23T20:32:34.6649290Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:32:34.6650390Z 100%|██████████| 1/1 [00:00<00:00, 84.49it/s]
2024-02-23T20:32:34.6813680Z 
2024-02-23T20:32:34.6933750Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:32:34.6935470Z 100%|██████████| 1/1 [00:00<00:00, 85.93it/s]
2024-02-23T20:32:34.6965170Z .
2024-02-23T20:32:34.6965910Z 0it [00:00, ?it/s]
2024-02-23T20:32:34.6966410Z 0it [00:00, ?it/s]
2024-02-23T20:32:34.7008320Z /Users/runner/miniconda3/envs/my-env/lib/python3.12/tarfile.py:2221: DeprecationWarning: Python 3.14 will, by default, filter extracted tar archives and reject files or modify their metadata. Use the filter argument to control this behavior.
2024-02-23T20:32:34.7011090Z   warnings.warn(
2024-02-23T20:32:34.7251830Z 
2024-02-23T20:32:34.7391550Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:32:34.7393380Z 100%|██████████| 1/1 [00:00<00:00, 73.45it/s]
2024-02-23T20:32:34.7542610Z 
2024-02-23T20:32:34.7665900Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:32:34.7667200Z 100%|██████████| 1/1 [00:00<00:00, 83.15it/s]
2024-02-23T20:32:34.7697540Z .
2024-02-23T20:32:34.7698860Z 0it [00:00, ?it/s]
2024-02-23T20:32:34.7699880Z 0it [00:00, ?it/s]
2024-02-23T20:32:34.7741520Z /Users/runner/miniconda3/envs/my-env/lib/python3.12/tarfile.py:2221: DeprecationWarning: Python 3.14 will, by default, filter extracted tar archives and reject files or modify their metadata. Use the filter argument to control this behavior.
2024-02-23T20:32:34.7745980Z   warnings.warn(
2024-02-23T20:32:34.8077450Z /Users/runner/miniconda3/envs/my-env/lib/python3.12/tarfile.py:2221: DeprecationWarning: Python 3.14 will, by default, filter extracted tar archives and reject files or modify their metadata. Use the filter argument to control this behavior.
2024-02-23T20:32:34.8080280Z   warnings.warn(
2024-02-23T20:32:34.8200660Z /Users/runner/miniconda3/envs/my-env/lib/python3.12/tarfile.py:2221: DeprecationWarning: Python 3.14 will, by default, filter extracted tar archives and reject files or modify their metadata. Use the filter argument to control this behavior.
2024-02-23T20:32:34.8203420Z   warnings.warn(
2024-02-23T20:32:34.8257980Z 
2024-02-23T20:32:34.8381860Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:32:34.8383020Z 100%|██████████| 1/1 [00:00<00:00, 82.61it/s]
2024-02-23T20:32:34.8419880Z 
2024-02-23T20:32:34.8475650Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:32:34.8476720Z 100%|██████████| 1/1 [00:00<00:00, 188.64it/s]
2024-02-23T20:32:34.8507080Z .
2024-02-23T20:32:34.8507570Z 0it [00:00, ?it/s]
2024-02-23T20:32:34.8508010Z 0it [00:00, ?it/s]
2024-02-23T20:32:34.8554320Z /Users/runner/miniconda3/envs/my-env/lib/python3.12/tarfile.py:2221: DeprecationWarning: Python 3.14 will, by default, filter extracted tar archives and reject files or modify their metadata. Use the filter argument to control this behavior.
2024-02-23T20:32:34.8557550Z   warnings.warn(
2024-02-23T20:32:34.8711480Z 
2024-02-23T20:32:34.8796160Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:32:34.8797700Z 100%|██████████| 1/1 [00:00<00:00, 124.05it/s]
2024-02-23T20:32:34.9049480Z /Users/runner/miniconda3/envs/my-env/lib/python3.12/tarfile.py:2221: DeprecationWarning: Python 3.14 will, by default, filter extracted tar archives and reject files or modify their metadata. Use the filter argument to control this behavior.
2024-02-23T20:32:34.9053540Z   warnings.warn(
2024-02-23T20:32:35.1110230Z 
2024-02-23T20:32:35.1417430Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:32:35.1427320Z 100%|██████████| 1/1 [00:00<00:00, 61.67it/s]
2024-02-23T20:32:35.1442000Z .
2024-02-23T20:32:35.1454280Z 0it [00:00, ?it/s]
2024-02-23T20:32:35.1472890Z 0it [00:00, ?it/s]
2024-02-23T20:32:35.1485630Z /Users/runner/miniconda3/envs/my-env/lib/python3.12/tarfile.py:2221: DeprecationWarning: Python 3.14 will, by default, filter extracted tar archives and reject files or modify their metadata. Use the filter argument to control this behavior.
2024-02-23T20:32:35.1494440Z   warnings.warn(
2024-02-23T20:32:35.1501570Z 
2024-02-23T20:32:35.1669770Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:32:35.1706660Z 100%|██████████| 1/1 [00:00<00:00, 62.17it/s]
2024-02-23T20:32:35.1713280Z .
2024-02-23T20:32:35.1747760Z 0it [00:00, ?it/s]
2024-02-23T20:32:35.1798580Z 0it [00:00, ?it/s]
2024-02-23T20:32:35.1833270Z /Users/runner/miniconda3/envs/my-env/lib/python3.12/tarfile.py:2221: DeprecationWarning: Python 3.14 will, by default, filter extracted tar archives and reject files or modify their metadata. Use the filter argument to control this behavior.
2024-02-23T20:32:35.1906050Z   warnings.warn(
2024-02-23T20:32:35.1964600Z /Users/runner/miniconda3/envs/my-env/lib/python3.12/tarfile.py:2221: DeprecationWarning: Python 3.14 will, by default, filter extracted tar archives and reject files or modify their metadata. Use the filter argument to control this behavior.
2024-02-23T20:32:35.1983100Z   warnings.warn(
2024-02-23T20:32:35.2919020Z 
2024-02-23T20:32:35.3382450Z   0%|          | 0/2 [00:00<?, ?it/s]
2024-02-23T20:32:35.3384210Z remove directory: /Users/runner/work/pyiron_base/pyiron_base/archive_folder/
2024-02-23T20:32:35.3418240Z 100%|██████████| 2/2 [00:00<00:00, 293.43it/s]
2024-02-23T20:32:35.3453620Z 
2024-02-23T20:32:35.3513660Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:32:35.3557670Z 100%|██████████| 1/1 [00:00<00:00, 57.13it/s]
2024-02-23T20:32:35.3563350Z .
2024-02-23T20:32:35.3592050Z   0%|          | 0/3 [00:00<?, ?it/s]
2024-02-23T20:32:35.3628020Z 100%|██████████| 3/3 [00:00<00:00, 120.67it/s]
2024-02-23T20:32:35.3669520Z 
2024-02-23T20:32:35.3684340Z 0it [00:00, ?it/s]
2024-02-23T20:32:35.3707760Z 0it [00:00, ?it/s]
2024-02-23T20:32:35.3741970Z 
2024-02-23T20:32:35.3779200Z 0it [00:00, ?it/s]
2024-02-23T20:32:35.3808320Z 0it [00:00, ?it/s]
2024-02-23T20:32:35.7763350Z s...............s/Users/runner/work/pyiron_base/pyiron_base/pyiron_base/database/filetable.py:138: FutureWarning: The behavior of DataFrame concatenation with empty or all-NA entries is deprecated. In a future version, this will no longer exclude empty or all-NA columns when determining the result dtypes. To retain the old behavior, exclude the relevant entries before the concat operation.
2024-02-23T20:32:35.7809730Z   self._job_table = pandas.concat(
2024-02-23T20:32:36.3524060Z ........s.......
2024-02-23T20:32:36.3525580Z EXCEPTION in get_items_sql:  (sqlite3.OperationalError) near "Wrong": syntax error
2024-02-23T20:32:36.3556320Z 0it [00:00, ?it/s]
2024-02-23T20:32:36.3557150Z [SQL: select * from simulation where A Wrong where Clause]
2024-02-23T20:32:36.3606880Z 0it [00:00, ?it/s]
2024-02-23T20:32:36.3608090Z (Background on this error at: https://sqlalche.me/e/20/e3q8)
2024-02-23T20:32:36.3640530Z The job toy_1 was saved and received the ID: 6
2024-02-23T20:32:36.3676310Z ['2.82e-02', '2.31e-05']
2024-02-23T20:32:36.3989880Z TestWithProject: Setting up test project
2024-02-23T20:32:36.5434800Z TestWithProject: Setting up test project
2024-02-23T20:32:36.5436500Z s...
2024-02-23T20:32:36.5439700Z The job test was saved and received the ID: test
2024-02-23T20:32:36.5442370Z 0it [00:00, ?it/s]
2024-02-23T20:32:36.5450450Z You have selected to start the job manually. To run it, go into the working directory /Users/runner/work/pyiron_base/pyiron_base/tests/database/test_no_database_project/test_hdf5/test and call 'python -m pyiron_base.cli wrapper -p /Users/runner/work/pyiron_base/pyiron_base/tests/database/test_no_database_project/test_hdf5/test -f /Users/runner/work/pyiron_base/pyiron_base/tests/database/test_no_database_project/test.h5/test ' 
2024-02-23T20:32:36.5461230Z 0it [00:00, ?it/s]
2024-02-23T20:32:37.6688500Z s...2024-02-23 20:32:37,579 - pyiron_log - WARNING - The job job_test is being loaded instead of running. To re-run use the argument 'delete_existing_job=True in create_job'
2024-02-23T20:32:38.2134880Z .2024-02-23 20:32:38,213 - pyiron_log - WARNING - The job job_test is being loaded instead of running. To re-run use the argument 'delete_existing_job=True in create_job'
2024-02-23T20:32:45.3532440Z .../Users/runner/miniconda3/envs/my-env/lib/python3.12/subprocess.py:1127: ResourceWarning: subprocess 6830 is still running
2024-02-23T20:32:45.3534640Z   _warn("subprocess %s is still running" % self.pid,
2024-02-23T20:32:45.3545850Z ResourceWarning: Enable tracemalloc to get the object allocation traceback
2024-02-23T20:32:45.3551780Z /Users/runner/miniconda3/envs/my-env/lib/python3.12/unittest/case.py:589: ResourceWarning: unclosed file <_io.BufferedReader name=22>
2024-02-23T20:32:45.3554400Z   if method() is not None:
2024-02-23T20:32:45.3556280Z ResourceWarning: Enable tracemalloc to get the object allocation traceback
2024-02-23T20:32:45.3559330Z /Users/runner/miniconda3/envs/my-env/lib/python3.12/unittest/case.py:589: ResourceWarning: unclosed file <_io.BufferedReader name=24>
2024-02-23T20:32:45.3561890Z   if method() is not None:
2024-02-23T20:32:45.3563700Z ResourceWarning: Enable tracemalloc to get the object allocation traceback
2024-02-23T20:32:45.3849180Z TestWithProject: Setting up test project
2024-02-23T20:32:45.3980800Z .
2024-02-23T20:32:45.3981330Z The job job_test was saved and received the ID: 6
2024-02-23T20:32:45.3988770Z The job job_no was saved and received the ID: 7
2024-02-23T20:32:45.3990480Z The job job_output_files was saved and received the ID: 8
2024-02-23T20:32:45.3994770Z The job job_sleep was saved and received the ID: 9
2024-02-23T20:32:45.3998840Z You have selected to start the job manually. To run it, go into the working directory /Users/runner/work/pyiron_base/pyiron_base/tests/flex/test_executablecontainer/job_sleep_hdf5/job_sleep and call 'python -m pyiron_base.cli wrapper -p /Users/runner/work/pyiron_base/pyiron_base/tests/flex/test_executablecontainer/job_sleep_hdf5/job_sleep -j 9 ' 
2024-02-23T20:32:45.4362790Z   0%|          | 0/4 [00:00<?, ?it/s]
2024-02-23T20:32:45.4363720Z 100%|██████████| 4/4 [00:00<00:00, 78.60it/s]
2024-02-23T20:32:45.6667960Z s/Users/runner/miniconda3/envs/my-env/lib/python3.12/site-packages/boltons/timeutils.py:441: DeprecationWarning: datetime.datetime.utcfromtimestamp() is deprecated and scheduled for removal in a future version. Use timezone-aware objects to represent datetimes in UTC: datetime.datetime.fromtimestamp(timestamp, datetime.UTC).
2024-02-23T20:32:45.6671620Z   EPOCH_NAIVE = datetime.utcfromtimestamp(0)
2024-02-23T20:33:03.5493380Z /Users/runner/work/pyiron_base/pyiron_base/pyiron_base/jobs/job/core.py:957: DeprecationWarning: Using __getitem__ on a job to access files in deprecated: use job.files instead!
2024-02-23T20:33:03.5511460Z   warnings.warn(
2024-02-23T20:33:03.5709730Z TestWithProject: Setting up test project
2024-02-23T20:33:03.5711340Z ..
2024-02-23T20:33:03.5711830Z The job job_conda_path was saved and received the ID: 6
2024-02-23T20:33:03.5871200Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:33:03.5875320Z 100%|██████████| 1/1 [00:00<00:00, 64.13it/s]
2024-02-23T20:33:03.7955760Z s2024-02-23 20:33:03,795 - pyiron_log - WARNING - The job my_function59f2ee8a4bfc7e75ebe938669f6a0c96 is being loaded instead of running. To re-run use the argument 'delete_existing_job=True in create_job'
2024-02-23T20:33:05.0397180Z TestWithProject: Setting up test project
2024-02-23T20:33:05.0398550Z The job my_function59f2ee8a4bfc7e75ebe938669f6a0c96 was saved and received the ID: 6
2024-02-23T20:33:05.0411960Z The job my_functiona55e17d1c7f6aba6df9475974051f26f was saved and received the ID: 7
2024-02-23T20:33:05.0413280Z The job my_functiona11fb2d5fe5c0352fcb600288ff480f3 was saved and received the ID: 8
2024-02-23T20:33:05.0414270Z The job my_function was saved and received the ID: 8
2024-02-23T20:33:05.0415240Z The job my_sleep_funct8348d547c48d0ed908e8d0116020d7ff was saved and received the ID: 8
2024-02-23T20:33:10.1880690Z ....2024-02-23 20:33:10,187 - pyiron_log - INFO - Job 8 intercept signal 15, job is shutting down
2024-02-23T20:33:11.4388460Z The job my_sleep_funct013175f3677ded5964e4a1f7249faaa9 was saved and received the ID: 9
2024-02-23T20:33:12.8063180Z The job my_sleep_funct0acba0b907be8f65b86f8b468cb39699 was saved and received the ID: 10
2024-02-23T20:33:14.5980950Z The job my_function_exe5b57f4c80ac3723d65f4c7936b92e38b was saved and received the ID: 11
2024-02-23T20:33:15.7732930Z ....
2024-02-23T20:33:15.8607420Z   0%|          | 0/6 [00:00<?, ?it/s]
2024-02-23T20:33:15.8612270Z 100%|██████████| 6/6 [00:00<00:00, 68.90it/s]
2024-02-23T20:33:23.4591180Z ss........................................................
2024-02-23T20:33:23.4592160Z TestWithProject: Setting up test project
2024-02-23T20:33:23.5309230Z 0it [00:00, ?it/s]
2024-02-23T20:33:23.5315820Z 0it [00:00, ?it/s]
2024-02-23T20:33:23.5344510Z ..ss..s./Users/runner/work/pyiron_base/pyiron_base/pyiron_base/utils/error.py:89: ImportWarning: Reduced functionality, since ['PIL', 'nbformat'] could not be imported.
2024-02-23T20:33:23.5346480Z   warnings.warn(self.message, category=ImportWarning)
2024-02-23T20:33:23.5347580Z ....
2024-02-23T20:33:23.5348660Z 0it [00:00, ?it/s]
2024-02-23T20:33:23.5349860Z 0it [00:00, ?it/s]
2024-02-23T20:33:23.5352580Z ./Users/runner/work/pyiron_base/pyiron_base/pyiron_base/utils/error.py:89: ImportWarning: Reduced functionality, since ['PIL', 'nbformat'] could not be imported.
2024-02-23T20:33:23.5354420Z   warnings.warn(self.message, category=ImportWarning)
2024-02-23T20:33:23.5357500Z ./Users/runner/work/pyiron_base/pyiron_base/pyiron_base/utils/error.py:89: ImportWarning: Reduced functionality, since ['PIL', 'nbformat'] could not be imported.
2024-02-23T20:33:23.5359330Z   warnings.warn(self.message, category=ImportWarning)
2024-02-23T20:33:23.8575100Z TestWithProject: Setting up test project
2024-02-23T20:33:23.8576050Z .....s.............................
2024-02-23T20:33:23.8577560Z 0it [00:00, ?it/s]
2024-02-23T20:33:23.8578080Z 0it [00:00, ?it/s]
2024-02-23T20:33:24.0467310Z TestWithProject: Setting up test project
2024-02-23T20:33:24.0467980Z ss.........
2024-02-23T20:33:24.0468550Z 0it [00:00, ?it/s]
2024-02-23T20:33:24.0468960Z 0it [00:00, ?it/s]
2024-02-23T20:33:24.0765320Z s..
2024-02-23T20:33:24.0765790Z TestWithProject: Setting up test project
2024-02-23T20:33:24.0766390Z 0it [00:00, ?it/s]
2024-02-23T20:33:24.0766770Z 0it [00:00, ?it/s]
2024-02-23T20:33:24.1424750Z ...
2024-02-23T20:33:24.1425230Z TestWithProject: Setting up test project
2024-02-23T20:33:24.1427180Z 0it [00:00, ?it/s]
2024-02-23T20:33:24.1428310Z 0it [00:00, ?it/s]
2024-02-23T20:33:24.2196200Z TestWithProject: Setting up test project
2024-02-23T20:33:24.2196820Z ...
2024-02-23T20:33:24.3198080Z 0it [00:00, ?it/s]
2024-02-23T20:33:24.4051640Z 0it [00:00, ?it/s]
2024-02-23T20:33:26.5183350Z s.s..s...........s
2024-02-23T20:33:26.5222300Z 0it [00:00, ?it/s]
2024-02-23T20:33:26.5333590Z 0it [00:00, ?it/s]
2024-02-23T20:33:28.2101230Z The job master was saved and received the ID: 6
2024-02-23T20:33:28.2102090Z .s..
2024-02-23T20:33:28.2105450Z The job child_1_1 was saved and received the ID: 7
2024-02-23T20:33:28.2107490Z The job child_1_2 was saved and received the ID: 8
2024-02-23T20:33:28.2110510Z The job child_1_3 was saved and received the ID: 9
2024-02-23T20:33:28.2112580Z The job child_1_4 was saved and received the ID: 10
2024-02-23T20:33:28.2114760Z The job child_1_5 was saved and received the ID: 11
2024-02-23T20:33:28.2118320Z The job child_1_6 was saved and received the ID: 12
2024-02-23T20:33:28.2120430Z The job child_1_7 was saved and received the ID: 13
2024-02-23T20:33:28.2122930Z The job child_1_8 was saved and received the ID: 14
2024-02-23T20:33:28.2125540Z The job child_1_9 was saved and received the ID: 15
2024-02-23T20:33:28.2127670Z The job master_2 was saved and received the ID: 16
2024-02-23T20:33:28.2129610Z The job child_2_1 was saved and received the ID: 17
2024-02-23T20:33:28.4654170Z   0%|          | 0/12 [00:00<?, ?it/s]
2024-02-23T20:33:28.4954850Z   8%|▊         | 1/12 [00:00<00:02,  3.93it/s]
2024-02-23T20:33:28.4956850Z 100%|██████████| 12/12 [00:00<00:00, 42.14it/s]
2024-02-23T20:33:28.4985470Z 
2024-02-23T20:33:28.4986960Z 0it [00:00, ?it/s]
2024-02-23T20:33:28.4988840Z 0it [00:00, ?it/s]
2024-02-23T20:33:30.6102160Z TestWithProject: Setting up test project
2024-02-23T20:33:30.6103310Z .....
2024-02-23T20:33:30.6104280Z The job job_script was saved and received the ID: 6
2024-02-23T20:33:30.6106010Z The job copy_test was saved and received the ID: 9
2024-02-23T20:33:30.7127320Z   0%|          | 0/3 [00:00<?, ?it/s]
2024-02-23T20:33:30.8038090Z 100%|██████████| 3/3 [00:00<00:00, 30.12it/s]
2024-02-23T20:33:30.8914010Z s...
2024-02-23T20:33:30.8915120Z The job job_test_run was saved and received the ID: 6
2024-02-23T20:33:30.8916580Z 0it [00:00, ?it/s]
2024-02-23T20:33:30.8917380Z 0it [00:00, ?it/s]
2024-02-23T20:33:32.5804460Z TestWithProject: Setting up test project
2024-02-23T20:33:32.5805940Z The job test was saved and received the ID: 6
2024-02-23T20:33:32.5807170Z s......
2024-02-23T20:33:32.5886050Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:33:32.5887260Z 100%|██████████| 1/1 [00:00<00:00, 134.17it/s]
2024-02-23T20:33:32.7438560Z .
2024-02-23T20:33:32.7439260Z 0it [00:00, ?it/s]
2024-02-23T20:33:32.7440000Z 0it [00:00, ?it/s]
2024-02-23T20:33:33.6283900Z s.....
2024-02-23T20:33:33.6285880Z TestWithProject: Setting up test project
2024-02-23T20:33:33.6333750Z 0it [00:00, ?it/s]
2024-02-23T20:33:33.6337520Z The job custom was saved and received the ID: 6
2024-02-23T20:33:33.6349720Z 0it [00:00, ?it/s]
2024-02-23T20:33:37.1758540Z TestWithProject: Setting up test project
2024-02-23T20:33:37.2121750Z The job toy_1 was saved and received the ID: 6
2024-02-23T20:33:37.2156890Z The job toy_2 was saved and received the ID: 7
2024-02-23T20:33:37.2541750Z The job toy_1 was saved and received the ID: 8
2024-02-23T20:33:37.2662740Z The job toy_2 was saved and received the ID: 9
2024-02-23T20:33:37.2715900Z The job toy_3 was saved and received the ID: 10
2024-02-23T20:33:37.2929760Z The job template was saved and received the ID: 11
2024-02-23T20:33:37.2967390Z The job template_copy was saved and received the ID: 12
2024-02-23T20:33:37.3055310Z The job job_single_debug was saved and received the ID: 12
2024-02-23T20:33:37.3209500Z The job job_with_executor_copy was saved and received the ID: 12
2024-02-23T20:33:38.9702490Z s..................................2024-02-23 20:33:38,969 - pyiron_log - INFO - job: job_with_executor_copy id: 12, status: submitted, run job (modal)
2024-02-23T20:33:38.9866770Z 2024-02-23 20:33:38,985 - pyiron_log - INFO - job: job_with_executor_copy id: 12, status: running, output: 
2024-02-23T20:33:38.9903110Z 2024-02-23 20:33:38,989 - pyiron_log - INFO - run job: job_with_executor_copy id: 12, status: collect
2024-02-23T20:33:39.0036220Z 2024-02-23 20:33:39,003 - pyiron_log - INFO - update master: None 12 executor
2024-02-23T20:33:39.4701350Z The job job_with_executor_run was saved and received the ID: 13
2024-02-23T20:33:41.1213810Z .2024-02-23 20:33:41,120 - pyiron_log - INFO - job: job_with_executor_run id: 13, status: submitted, run job (modal)
2024-02-23T20:33:41.1415810Z 2024-02-23 20:33:41,139 - pyiron_log - INFO - job: job_with_executor_run id: 13, status: running, output: 
2024-02-23T20:33:41.1455480Z 2024-02-23 20:33:41,142 - pyiron_log - INFO - run job: job_with_executor_run id: 13, status: collect
2024-02-23T20:33:41.2402050Z 2024-02-23 20:33:41,154 - pyiron_log - INFO - update master: None 13 executor
2024-02-23T20:33:41.2670610Z The job job_with_executor_wait was saved and received the ID: 14
2024-02-23T20:33:43.1442620Z .2024-02-23 20:33:43,141 - pyiron_log - INFO - job: job_with_executor_wait id: 14, status: submitted, run job (modal)
2024-02-23T20:33:43.1641280Z 2024-02-23 20:33:43,163 - pyiron_log - INFO - job: job_with_executor_wait id: 14, status: running, output: 
2024-02-23T20:33:43.1691860Z 2024-02-23 20:33:43,167 - pyiron_log - INFO - run job: job_with_executor_wait id: 14, status: collect
2024-02-23T20:33:43.1826020Z 2024-02-23 20:33:43,181 - pyiron_log - INFO - update master: None 14 executor
2024-02-23T20:33:43.2994250Z .../Users/runner/work/pyiron_base/pyiron_base/tests/job/test_genericJob.py:157: DeprecationWarning: pyiron_base.jobs.job.core.list_files is deprecated: use job.files.list().
2024-02-23T20:33:43.3172300Z   ham.list_files(), ["test_file"]
2024-02-23T20:33:43.3245700Z /Users/runner/miniconda3/envs/my-env/lib/python3.12/tarfile.py:2221: DeprecationWarning: Python 3.14 will, by default, filter extracted tar archives and reject files or modify their metadata. Use the filter argument to control this behavior.
2024-02-23T20:33:43.3374280Z   warnings.warn(
2024-02-23T20:33:44.1215740Z ...
2024-02-23T20:33:44.1216290Z 0it [00:00, ?it/s]
2024-02-23T20:33:44.1216910Z 0it [00:00, ?it/s]
2024-02-23T20:33:44.1296170Z 
2024-02-23T20:33:44.1297840Z 0it [00:00, ?it/s]
2024-02-23T20:33:44.1298720Z 0it [00:00, ?it/s]
2024-02-23T20:33:44.2989790Z .......2024-02-23 20:33:44,298 - pyiron_log - WARNING - Job 'job_without_hdf5' does not exist and cannot be loaded
2024-02-23T20:33:45.5197550Z .../Users/runner/miniconda3/envs/my-env/lib/python3.12/tarfile.py:2221: DeprecationWarning: Python 3.14 will, by default, filter extracted tar archives and reject files or modify their metadata. Use the filter argument to control this behavior.
2024-02-23T20:33:45.5206380Z   warnings.warn(
2024-02-23T20:33:45.5239060Z /Users/runner/work/pyiron_base/pyiron_base/tests/job/test_genericJob.py:512: DeprecationWarning: pyiron_base.jobs.job.core.list_files is deprecated: use job.files.list().
2024-02-23T20:33:45.5276580Z   wd_files = job_restart.list_files()
2024-02-23T20:33:45.7532180Z .2024-02-23 20:33:45,752 - pyiron_log - WARNING - Job aborted
2024-02-23T20:33:45.7534270Z 2024-02-23 20:33:45,753 - pyiron_log - WARNING - 
2024-02-23T20:33:46.0195730Z 2024-02-23 20:33:46,018 - pyiron_log - WARNING - Job aborted
2024-02-23T20:33:46.0197560Z 2024-02-23 20:33:46,019 - pyiron_log - WARNING - 
2024-02-23T20:33:46.6886830Z ....................../Users/runner/miniconda3/envs/my-env/lib/python3.12/tarfile.py:2221: DeprecationWarning: Python 3.14 will, by default, filter extracted tar archives and reject files or modify their metadata. Use the filter argument to control this behavior.
2024-02-23T20:33:46.7883630Z   warnings.warn(
2024-02-23T20:33:46.7884320Z The job job_without_hdf5 was saved and received the ID: 16
2024-02-23T20:33:46.8788020Z The job toy_1_restart was saved and received the ID: 16
2024-02-23T20:33:46.8884510Z .......
2024-02-23T20:33:46.9791100Z The job success_0 was saved and received the ID: 17
2024-02-23T20:33:46.9886540Z   0%|          | 0/16 [00:00<?, ?it/s]
2024-02-23T20:33:47.0793370Z The job aborted_1 was saved and received the ID: 18
2024-02-23T20:33:47.1795330Z The job success_1 was saved and received the ID: 19
2024-02-23T20:33:47.2718430Z  44%|████▍     | 7/16 [00:00<00:00, 69.49it/s]
2024-02-23T20:33:47.2719250Z The job aborted_2 was saved and received the ID: 20
2024-02-23T20:33:47.2776740Z  88%|████████▊ | 14/16 [00:00<00:00, 28.78it/s]
2024-02-23T20:33:47.2778850Z The job rerun_aborted was saved and received the ID: 21
2024-02-23T20:33:47.2787440Z 100%|██████████| 16/16 [00:00<00:00, 34.20it/s]
2024-02-23T20:33:47.2788260Z The job rerun_aborted was saved and received the ID: 21
2024-02-23T20:33:47.4720110Z s...
2024-02-23T20:33:47.4721000Z The job job_test_run was saved and received the ID: 6
2024-02-23T20:33:47.4722210Z 0it [00:00, ?it/s]
2024-02-23T20:33:47.4723140Z 0it [00:00, ?it/s]
2024-02-23T20:33:47.6838290Z The job job_test_run was saved and received the ID: 6
2024-02-23T20:33:47.6855950Z s....
2024-02-23T20:33:47.6907830Z 0it [00:00, ?it/s]
2024-02-23T20:33:47.6947550Z 0it [00:00, ?it/s]
2024-02-23T20:33:47.8557360Z TestWithProject: Setting up test project
2024-02-23T20:33:47.8558500Z s.................
2024-02-23T20:33:47.8738040Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:33:47.8739440Z 100%|██████████| 1/1 [00:00<00:00, 57.14it/s]
2024-02-23T20:33:47.9619880Z ss.....s.......2024-02-23 20:33:47,961 - pyiron_log - WARNING - Job 'script' does not exist and cannot be loaded
2024-02-23T20:33:47.9623700Z 2024-02-23 20:33:47,961 - pyiron_log - WARNING - Job 'script' does not exist and could not be removed
2024-02-23T20:33:48.0577000Z .2024-02-23 20:33:48,057 - pyiron_log - WARNING - Job 'script' does not exist and cannot be loaded
2024-02-23T20:33:48.0581230Z 2024-02-23 20:33:48,057 - pyiron_log - WARNING - Job 'script' does not exist and could not be removed
2024-02-23T20:33:49.5064650Z TestWithProject: Setting up test project
2024-02-23T20:33:49.5135270Z .
2024-02-23T20:33:49.5135880Z The job script was saved and received the ID: 6
2024-02-23T20:33:49.5138120Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:33:49.5138800Z The job script was saved and received the ID: 6
2024-02-23T20:33:49.5141290Z 100%|██████████| 1/1 [00:00<00:00, 208.11it/s]
2024-02-23T20:33:49.5689440Z 2024-02-23 20:33:49,568 - pyiron_log - WARNING - Job 'script' does not exist and cannot be loaded
2024-02-23T20:33:49.5693560Z 2024-02-23 20:33:49,568 - pyiron_log - WARNING - Job 'script' does not exist and could not be removed
2024-02-23T20:33:51.0177100Z The job script was saved and received the ID: 6
2024-02-23T20:33:51.0180360Z .
2024-02-23T20:33:51.0231330Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:33:51.0233060Z 100%|██████████| 1/1 [00:00<00:00, 209.65it/s]
2024-02-23T20:33:51.2178560Z 2024-02-23 20:33:51,217 - pyiron_log - WARNING - Job 'script' does not exist and cannot be loaded
2024-02-23T20:33:51.2180390Z 2024-02-23 20:33:51,217 - pyiron_log - WARNING - Job 'script' does not exist and could not be removed
2024-02-23T20:33:51.4015860Z The job script was saved and received the ID: 6
2024-02-23T20:33:51.4016680Z .
2024-02-23T20:33:51.4060680Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:33:51.4061670Z 100%|██████████| 1/1 [00:00<00:00, 244.18it/s]
2024-02-23T20:33:51.4348360Z 2024-02-23 20:33:51,434 - pyiron_log - WARNING - Job 'script' does not exist and cannot be loaded
2024-02-23T20:33:51.4350100Z 2024-02-23 20:33:51,434 - pyiron_log - WARNING - Job 'script' does not exist and could not be removed
2024-02-23T20:33:51.4577700Z .
2024-02-23T20:33:51.4578760Z 0it [00:00, ?it/s]
2024-02-23T20:33:51.4579750Z 0it [00:00, ?it/s]
2024-02-23T20:33:51.4652520Z s
2024-02-23T20:33:51.4653010Z TestWithProject: Setting up test project
2024-02-23T20:33:51.4653860Z 0it [00:00, ?it/s]
2024-02-23T20:33:51.4655200Z 0it [00:00, ?it/s]
2024-02-23T20:33:51.5933600Z The job runner was saved and received the ID: 6
2024-02-23T20:34:06.8844710Z The job script was saved and received the ID: 7
2024-02-23T20:34:06.8845360Z ..
2024-02-23T20:34:06.8848770Z You have selected to start the job manually. To run it, go into the working directory /Users/runner/work/pyiron_base/pyiron_base/tests/job/test_worker/sub/script_hdf5/script and call 'python -m pyiron_base.cli wrapper -p /Users/runner/work/pyiron_base/pyiron_base/tests/job/test_worker/sub/script_hdf5/script -j 7 ' 
2024-02-23T20:34:06.8851890Z 0it [00:00, ?it/s]
2024-02-23T20:34:06.8852390Z 0it [00:00, ?it/s]
2024-02-23T20:34:07.0968490Z s..
2024-02-23T20:34:07.0970520Z 0it [00:00, ?it/s]
2024-02-23T20:34:07.0970890Z 0it [00:00, ?it/s]
2024-02-23T20:34:13.6842040Z remove directory: /Users/runner/work/pyiron_base/pyiron_base/tests/master/master_copy/
2024-02-23T20:34:13.6843620Z ....
2024-02-23T20:34:13.6853090Z TestWithProject: Setting up test project
2024-02-23T20:34:13.6854650Z The job master_toy was saved and received the ID: 6
2024-02-23T20:34:13.6855440Z The job test_0 was saved and received the ID: 7
2024-02-23T20:34:13.6856220Z The job test_1 was saved and received the ID: 8
2024-02-23T20:34:13.6856970Z The job test_2 was saved and received the ID: 9
2024-02-23T20:34:13.6857740Z The job test_3 was saved and received the ID: 10
2024-02-23T20:34:13.6858490Z The job test_4 was saved and received the ID: 11
2024-02-23T20:34:13.6859250Z The job test_5 was saved and received the ID: 12
2024-02-23T20:34:13.6860520Z The job test_6 was saved and received the ID: 13
2024-02-23T20:34:13.6861400Z The job test_7 was saved and received the ID: 14
2024-02-23T20:34:13.6862210Z The job test_8 was saved and received the ID: 15
2024-02-23T20:34:13.6863740Z The job test_9 was saved and received the ID: 16
2024-02-23T20:34:13.9052300Z   0%|          | 0/11 [00:00<?, ?it/s]
2024-02-23T20:34:13.9108010Z   9%|▉         | 1/11 [00:00<00:02,  4.54it/s]
2024-02-23T20:34:13.9109500Z 100%|██████████| 11/11 [00:00<00:00, 48.65it/s]
2024-02-23T20:34:13.9172140Z ss
2024-02-23T20:34:13.9173560Z 0it [00:00, ?it/s]
2024-02-23T20:34:13.9174800Z 0it [00:00, ?it/s]
2024-02-23T20:34:13.9484200Z .
2024-02-23T20:34:13.9485130Z 0it [00:00, ?it/s]
2024-02-23T20:34:13.9486740Z 0it [00:00, ?it/s]
2024-02-23T20:34:13.9883710Z .
2024-02-23T20:34:13.9887840Z 0it [00:00, ?it/s]
2024-02-23T20:34:13.9889720Z 0it [00:00, ?it/s]
2024-02-23T20:34:14.1035000Z .
2024-02-23T20:34:14.1036450Z 0it [00:00, ?it/s]
2024-02-23T20:34:14.1037110Z 0it [00:00, ?it/s]
2024-02-23T20:34:14.1384380Z .s......
2024-02-23T20:34:14.1386260Z 0it [00:00, ?it/s]
2024-02-23T20:34:14.1400160Z 0it [00:00, ?it/s]
2024-02-23T20:34:16.2140260Z TestWithProject: Setting up test project
2024-02-23T20:34:16.2141210Z ....
2024-02-23T20:34:16.2141780Z The job toy_1 was saved and received the ID: 6
2024-02-23T20:34:16.2142500Z The job toy_2 was saved and received the ID: 7
2024-02-23T20:34:16.2143190Z The job toy_1 was saved and received the ID: 8
2024-02-23T20:34:16.2143860Z The job toy_2 was saved and received the ID: 9
2024-02-23T20:34:16.2144550Z The job toy_3 was saved and received the ID: 10
2024-02-23T20:34:16.2649370Z   0%|          | 0/5 [00:00<?, ?it/s]
2024-02-23T20:34:16.2650230Z 100%|██████████| 5/5 [00:00<00:00, 99.03it/s]
2024-02-23T20:34:17.8144540Z s./Users/runner/work/pyiron_base/pyiron_base/pyiron_base/project/maintenance.py:291: UserWarning: 
2024-02-23T20:34:17.8146260Z                 The database statistics is only available for a Postgresql database
2024-02-23T20:34:17.8147170Z                 
2024-02-23T20:34:17.8147570Z   warnings.warn(
2024-02-23T20:34:17.8202150Z TestWithProject: Setting up test project
2024-02-23T20:34:17.8203010Z 
2024-02-23T20:34:17.8203440Z The job toy_1 was saved and received the ID: 6
2024-02-23T20:34:17.8204260Z The job toy_2 was saved and received the ID: 7
2024-02-23T20:34:17.8204980Z The job toy_1 was saved and received the ID: 8
2024-02-23T20:34:17.8205700Z The job toy_2 was saved and received the ID: 9
2024-02-23T20:34:17.8206440Z The job toy_3 was saved and received the ID: 10
2024-02-23T20:34:17.8217400Z   0%|          | 0/5 [00:00<?, ?it/s]2024-02-23 20:34:17,821 - pyiron_log - WARNING - Specifying job_name is deprecated and ignored! Future versions will change signature.
2024-02-23T20:34:18.0422210Z 
2024-02-23T20:34:18.0440000Z  20%|██        | 1/5 [00:00<00:00,  4.51it/s]2024-02-23 20:34:18,043 - pyiron_log - WARNING - Specifying job_name is deprecated and ignored! Future versions will change signature.
2024-02-23T20:34:18.3220780Z 
2024-02-23T20:34:18.3255130Z  40%|████      | 2/5 [00:00<00:00,  4.20it/s]2024-02-23 20:34:18,293 - pyiron_log - WARNING - Specifying job_name is deprecated and ignored! Future versions will change signature.
2024-02-23T20:34:18.5075420Z 
2024-02-23T20:34:18.5090150Z  60%|██████    | 3/5 [00:00<00:00,  4.39it/s]2024-02-23 20:34:18,508 - pyiron_log - WARNING - Specifying job_name is deprecated and ignored! Future versions will change signature.
2024-02-23T20:34:18.6969980Z 
2024-02-23T20:34:18.6985200Z  80%|████████  | 4/5 [00:00<00:00,  4.70it/s]2024-02-23 20:34:18,698 - pyiron_log - WARNING - Specifying job_name is deprecated and ignored! Future versions will change signature.
2024-02-23T20:34:19.0151520Z 
2024-02-23T20:34:19.0817530Z 100%|██████████| 5/5 [00:01<00:00,  4.51it/s]
2024-02-23T20:34:19.0823100Z 100%|██████████| 5/5 [00:01<00:00,  4.49it/s]
2024-02-23T20:34:19.8483780Z ..
2024-02-23T20:34:19.8967590Z   0%|          | 0.00/204k [00:00<?, ?B/s]2024-02-23 20:34:19,861 - pyiron_log - WARNING - Specifying job_name is deprecated and ignored! Future versions will change signature.
2024-02-23T20:34:20.1268090Z 
2024-02-23T20:34:20.1598890Z  29%|██▉       | 59.7k/204k [00:00<00:00, 215kB/s]
2024-02-23T20:34:20.1599900Z 100%|██████████| 204k/204k [00:00<00:00, 657kB/s] 
2024-02-23T20:34:20.5198310Z Total Jobs: 5	Errors: 0	Skipped (no bug detected): 4
2024-02-23T20:34:20.5201530Z .
2024-02-23T20:34:20.5319910Z   0%|          | 0.00/196k [00:00<?, ?B/s]2024-02-23 20:34:20,531 - pyiron_log - WARNING - Specifying job_name is deprecated and ignored! Future versions will change signature.
2024-02-23T20:34:20.7218870Z 
2024-02-23T20:34:20.7544980Z  26%|██▌       | 51.5k/196k [00:00<00:00, 256kB/s]
2024-02-23T20:34:20.7545980Z 100%|██████████| 196k/196k [00:00<00:00, 838kB/s] 
2024-02-23T20:34:21.1096770Z Total Jobs: 5	Errors: 0	Skipped (no bug detected): 4
2024-02-23T20:34:21.1097550Z .
2024-02-23T20:34:21.1204910Z   0%|          | 0.00/196k [00:00<?, ?B/s]2024-02-23 20:34:21,120 - pyiron_log - WARNING - Specifying job_name is deprecated and ignored! Future versions will change signature.
2024-02-23T20:34:21.2979280Z 
2024-02-23T20:34:21.3420930Z  26%|██▌       | 51.5k/196k [00:00<00:00, 274kB/s]
2024-02-23T20:34:21.3423120Z 100%|██████████| 196k/196k [00:00<00:00, 846kB/s] 
2024-02-23T20:34:21.7497930Z Total Jobs: 5	Errors: 0	Skipped (no bug detected): 4
2024-02-23T20:34:21.7498900Z .
2024-02-23T20:34:21.7620980Z   0%|          | 0.00/196k [00:00<?, ?B/s]2024-02-23 20:34:21,761 - pyiron_log - WARNING - Specifying job_name is deprecated and ignored! Future versions will change signature.
2024-02-23T20:34:21.9510320Z 
2024-02-23T20:34:21.9835340Z  26%|██▌       | 51.5k/196k [00:00<00:00, 257kB/s]
2024-02-23T20:34:21.9836610Z 100%|██████████| 196k/196k [00:00<00:00, 842kB/s] 
2024-02-23T20:34:22.1346670Z Total Jobs: 5	Errors: 0	Skipped (no bug detected): 4
2024-02-23T20:34:22.1347700Z .
2024-02-23T20:34:22.2188730Z   0%|          | 0/5 [00:00<?, ?it/s]
2024-02-23T20:34:22.3191810Z 100%|██████████| 5/5 [00:00<00:00, 59.71it/s]
2024-02-23T20:34:22.3475690Z ss
2024-02-23T20:34:22.3584930Z 0it [00:00, ?it/s]
2024-02-23T20:34:22.3633540Z 0it [00:00, ?it/s]
2024-02-23T20:34:22.3635120Z .
2024-02-23T20:34:22.3638140Z 0it [00:00, ?it/s]
2024-02-23T20:34:22.3641380Z 0it [00:00, ?it/s]
2024-02-23T20:34:22.3644600Z .
2024-02-23T20:34:22.3647750Z 0it [00:00, ?it/s]
2024-02-23T20:34:22.3650960Z 0it [00:00, ?it/s]
2024-02-23T20:34:22.3654170Z .
2024-02-23T20:34:22.3657500Z 0it [00:00, ?it/s]
2024-02-23T20:34:22.3660700Z 0it [00:00, ?it/s]
2024-02-23T20:34:22.3663100Z 
2024-02-23T20:34:22.3667010Z 0it [00:00, ?it/s]
2024-02-23T20:34:22.3672140Z 0it [00:00, ?it/s]
2024-02-23T20:34:22.3684920Z 
2024-02-23T20:34:22.3687270Z 0it [00:00, ?it/s]
2024-02-23T20:34:22.3692930Z 0it [00:00, ?it/s]
2024-02-23T20:34:24.0015820Z TestWithProject: Setting up test project
2024-02-23T20:34:24.0585510Z ....
2024-02-23T20:34:24.0586090Z The job toy_1 was saved and received the ID: 6
2024-02-23T20:34:24.0633960Z The job toy_2 was saved and received the ID: 7
2024-02-23T20:34:24.0718660Z The job toy_1 was saved and received the ID: 8
2024-02-23T20:34:24.0755930Z The job toy_2 was saved and received the ID: 9
2024-02-23T20:34:24.0783680Z The job toy_3 was saved and received the ID: 10
2024-02-23T20:34:24.1656070Z   0%|          | 0/5 [00:00<?, ?it/s]
2024-02-23T20:34:24.3393110Z  20%|██        | 1/5 [00:00<00:00,  6.79it/s]
2024-02-23T20:34:24.4537510Z  40%|████      | 2/5 [00:00<00:00,  5.80it/s]
2024-02-23T20:34:24.5646300Z  60%|██████    | 3/5 [00:00<00:00,  6.86it/s]
2024-02-23T20:34:24.6743300Z  80%|████████  | 4/5 [00:00<00:00,  7.57it/s]
2024-02-23T20:34:24.6746930Z 100%|██████████| 5/5 [00:00<00:00,  8.06it/s]
2024-02-23T20:34:24.6748370Z 100%|██████████| 5/5 [00:00<00:00,  7.44it/s]
2024-02-23T20:34:24.6791010Z 
2024-02-23T20:34:24.6792400Z 0it [00:00, ?it/s]
2024-02-23T20:34:24.6793170Z 0it [00:00, ?it/s]
2024-02-23T20:34:24.6830990Z 
2024-02-23T20:34:24.8046830Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:34:24.8185320Z 100%|██████████| 1/1 [00:00<00:00,  8.24it/s]
2024-02-23T20:34:24.8303220Z 100%|██████████| 1/1 [00:00<00:00,  8.22it/s]
2024-02-23T20:34:25.2099350Z The job toy1 was saved and received the ID: toy1
2024-02-23T20:34:25.2100250Z .
2024-02-23T20:34:25.2100670Z The job toy2 was saved and received the ID: toy2
2024-02-23T20:34:25.3241110Z   0%|          | 0/2 [00:00<?, ?it/s]
2024-02-23T20:34:25.4756590Z  50%|█████     | 1/2 [00:00<00:00,  8.80it/s]
2024-02-23T20:34:25.4758230Z 100%|██████████| 2/2 [00:00<00:00,  7.36it/s]
2024-02-23T20:34:25.4759470Z 100%|██████████| 2/2 [00:00<00:00,  7.53it/s]
2024-02-23T20:34:25.4945840Z 
2024-02-23T20:34:25.6871760Z   0%|          | 0/2 [00:00<?, ?it/s]
2024-02-23T20:34:25.6969110Z 100%|██████████| 2/2 [00:00<00:00, 16.14it/s]
2024-02-23T20:34:25.6994050Z 100%|██████████| 2/2 [00:00<00:00, 15.98it/s]
2024-02-23T20:34:25.7061230Z 
2024-02-23T20:34:25.7080960Z   0%|          | 0/2 [00:00<?, ?it/s]
2024-02-23T20:34:25.7205180Z 100%|██████████| 2/2 [00:00<00:00, 39.36it/s]
2024-02-23T20:34:25.7619530Z ../Users/runner/work/pyiron_base/pyiron_base/pyiron_base/project/maintenance.py:291: UserWarning: 
2024-02-23T20:34:25.7626940Z                 The database statistics is only available for a Postgresql database
2024-02-23T20:34:25.7630090Z                 
2024-02-23T20:34:25.7636940Z   warnings.warn(
2024-02-23T20:34:26.5647010Z ...
2024-02-23T20:34:26.6453570Z   0%|          | 0/5 [00:00<?, ?it/s]
2024-02-23T20:34:26.6455470Z 100%|██████████| 5/5 [00:00<00:00, 62.10it/s]
2024-02-23T20:34:29.4985290Z ..
2024-02-23T20:34:29.4986550Z TestWithProject: Setting up test project
2024-02-23T20:34:29.5189330Z The job toy_1 was saved and received the ID: 6
2024-02-23T20:34:29.5190190Z The job toy_2 was saved and received the ID: 7
2024-02-23T20:34:29.5191550Z The job toy_1 was saved and received the ID: 8
2024-02-23T20:34:29.5193340Z The job toy_2 was saved and received the ID: 9
2024-02-23T20:34:29.5195480Z The job toy_3 was saved and received the ID: 10
2024-02-23T20:34:29.6554310Z   0%|          | 0/5 [00:00<?, ?it/s]
2024-02-23T20:34:29.7395720Z  80%|████████  | 4/5 [00:00<00:00, 25.57it/s]
2024-02-23T20:34:29.7399350Z 100%|██████████| 5/5 [00:00<00:00, 21.00it/s]
2024-02-23T20:34:29.8304200Z ..
2024-02-23T20:34:29.8304940Z TestWithProject: Setting up test project
2024-02-23T20:34:29.8306180Z 0it [00:00, ?it/s]
2024-02-23T20:34:29.8306730Z 0it [00:00, ?it/s]
2024-02-23T20:34:29.9172900Z sss.........s.......s..........s...s..2024-02-23 20:34:29,916 - pyiron_log - WARNING - Here is a warning
2024-02-23T20:34:30.0215110Z .................2024-02-23 20:34:30,021 - pyiron_log - WARNING - Credentials file can be read by other users - check permissions.
2024-02-23T20:34:31.9432530Z TestWithProject: Setting up test project
2024-02-23T20:34:31.9436210Z ....s......................................
2024-02-23T20:34:31.9452790Z 0it [00:00, ?it/s]
2024-02-23T20:34:31.9454420Z 0it [00:00, ?it/s]
2024-02-23T20:34:32.7675630Z s.........
2024-02-23T20:34:32.7676400Z TestWithProject: Setting up test project
2024-02-23T20:34:32.7681800Z 0it [00:00, ?it/s]
2024-02-23T20:34:32.7682570Z 0it [00:00, ?it/s]
2024-02-23T20:34:33.4253140Z s.........../Users/runner/work/pyiron_base/pyiron_base/pyiron_base/project/maintenance.py:291: UserWarning: 
2024-02-23T20:34:33.4254920Z                 The database statistics is only available for a Postgresql database
2024-02-23T20:34:33.4255990Z                 
2024-02-23T20:34:33.4256440Z   warnings.warn(
2024-02-23T20:34:33.4336360Z TestWithProject: Setting up test project
2024-02-23T20:34:33.4337160Z 
2024-02-23T20:34:33.4337460Z The job test_job was saved and received the ID: 6
2024-02-23T20:34:33.4847760Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:34:33.4848790Z 100%|██████████| 1/1 [00:00<00:00, 19.78it/s]
2024-02-23T20:34:33.5170780Z 
2024-02-23T20:34:33.5171560Z 0it [00:00, ?it/s]
2024-02-23T20:34:33.5172190Z 0it [00:00, ?it/s]
2024-02-23T20:34:33.5230690Z 
2024-02-23T20:34:33.5231650Z 0it [00:00, ?it/s]
2024-02-23T20:34:33.5232830Z 0it [00:00, ?it/s]
2024-02-23T20:34:33.8190810Z .
2024-02-23T20:34:33.8350580Z   0%|          | 0/1 [00:00<?, ?it/s]
2024-02-23T20:34:33.8353030Z 100%|██████████| 1/1 [00:00<00:00, 63.82it/s]
2024-02-23T20:34:35.0785790Z TestWithProject: Setting up test project
2024-02-23T20:34:35.0787520Z s
2024-02-23T20:34:35.0788300Z The job test_a was saved and received the ID: 6
2024-02-23T20:34:35.0789870Z The job test_b was saved and received the ID: 7
2024-02-23T20:34:35.0791530Z The job test_c was saved and received the ID: 8
2024-02-23T20:34:35.0793160Z The job test_d was saved and received the ID: 9
2024-02-23T20:34:35.0794860Z The job test_table was saved and received the ID: 10
2024-02-23T20:34:35.0854620Z Loading and filtering jobs:   0%|          | 0/5 [00:00<?, ?it/s]
2024-02-23T20:34:35.0857160Z Loading and filtering jobs: 100%|██████████| 5/5 [00:00<00:00, 787.90it/s]
2024-02-23T20:34:35.0868370Z 
2024-02-23T20:34:35.0882410Z   0%|          | 0/2 [00:00<?, ?it/s]
2024-02-23T20:34:35.0883980Z 100%|██████████| 2/2 [00:00<00:00, 1764.91it/s]
2024-02-23T20:34:35.2535520Z /Users/runner/work/pyiron_base/pyiron_base/pyiron_base/jobs/datamining.py:651: PerformanceWarning: 
2024-02-23T20:34:35.3540660Z your performance may suffer as PyTables will pickle object types that it cannot
2024-02-23T20:34:35.3619240Z map directly to c-types [inferred_type->mixed,key->block1_values] [items->Index(['name', 'array'], dtype='object')]
2024-02-23T20:34:35.4618940Z 
2024-02-23T20:34:35.4619670Z The job test_table was saved and received the ID: 10
2024-02-23T20:34:35.5508500Z   self.pyiron_table._df.to_hdf(
2024-02-23T20:34:35.5551940Z .
2024-02-23T20:34:35.5556800Z Loading and filtering jobs:   0%|          | 0/5 [00:00<?, ?it/s]
2024-02-23T20:34:35.5561170Z Loading and filtering jobs: 100%|██████████| 5/5 [00:00<00:00, 528.94it/s]
2024-02-23T20:34:35.5565100Z 
2024-02-23T20:34:35.5569740Z   0%|          | 0/2 [00:00<?, ?it/s]
2024-02-23T20:34:35.5575330Z 100%|██████████| 2/2 [00:00<00:00, 1378.35it/s]
2024-02-23T20:34:35.6041540Z /Users/runner/work/pyiron_base/pyiron_base/pyiron_base/jobs/datamining.py:651: PerformanceWarning: 
2024-02-23T20:34:35.6045020Z your performance may suffer as PyTables will pickle object types that it cannot
2024-02-23T20:34:35.6050450Z map directly to c-types [inferred_type->mixed,key->block1_values] [items->Index(['name', 'array'], dtype='object')]
2024-02-23T20:34:35.6053790Z 
2024-02-23T20:34:35.6055760Z   self.pyiron_table._df.to_hdf(
2024-02-23T20:34:35.9996420Z .
2024-02-23T20:34:35.9997270Z The job test_table was saved and received the ID: 10
2024-02-23T20:34:35.9999410Z Loading and filtering jobs:   0%|          | 0/5 [00:00<?, ?it/s]
2024-02-23T20:34:36.0005520Z Loading and filtering jobs: 100%|██████████| 5/5 [00:00<00:00, 533.04it/s]
2024-02-23T20:34:36.0007720Z 
2024-02-23T20:34:36.0009280Z   0%|          | 0/2 [00:00<?, ?it/s]
2024-02-23T20:34:36.0012870Z 100%|██████████| 2/2 [00:00<00:00, 1100.00it/s]
2024-02-23T20:34:36.1793440Z The job test_table was saved and received the ID: 10
2024-02-23T20:34:36.1794640Z .
2024-02-23T20:34:36.1848710Z Loading and filtering jobs:   0%|          | 0/5 [00:00<?, ?it/s]
2024-02-23T20:34:36.1851000Z Loading and filtering jobs: 100%|██████████| 5/5 [00:00<00:00, 995.42it/s]
2024-02-23T20:34:36.1860680Z 
2024-02-23T20:34:36.1872370Z   0%|          | 0/2 [00:00<?, ?it/s]
2024-02-23T20:34:36.1874020Z 100%|██████████| 2/2 [00:00<00:00, 2190.81it/s]
2024-02-23T20:34:36.5068690Z .
2024-02-23T20:34:36.5069560Z The job test_table was saved and received the ID: 10
2024-02-23T20:34:36.5122160Z Loading and filtering jobs:   0%|          | 0/5 [00:00<?, ?it/s]
2024-02-23T20:34:36.5124120Z Loading and filtering jobs: 100%|██████████| 5/5 [00:00<00:00, 972.57it/s]
2024-02-23T20:34:36.5135480Z 
2024-02-23T20:34:36.5146340Z   0%|          | 0/2 [00:00<?, ?it/s]
2024-02-23T20:34:36.5147350Z 100%|██████████| 2/2 [00:00<00:00, 2167.04it/s]
2024-02-23T20:34:36.5643450Z /Users/runner/work/pyiron_base/pyiron_base/pyiron_base/jobs/datamining.py:651: PerformanceWarning: 
2024-02-23T20:34:36.5646680Z your performance may suffer as PyTables will pickle object types that it cannot
2024-02-23T20:34:36.5650520Z map directly to c-types [inferred_type->mixed,key->block1_values] [items->Index(['name', 'array'], dtype='object')]
2024-02-23T20:34:36.5652750Z 
2024-02-23T20:34:36.5653120Z   self.pyiron_table._df.to_hdf(
2024-02-23T20:34:36.7624110Z .
2024-02-23T20:34:36.8236880Z   0%|          | 0/4 [00:00<?, ?it/s]
2024-02-23T20:34:36.8238530Z 100%|██████████| 4/4 [00:00<00:00, 65.52it/s]
2024-02-23T20:34:38.0881480Z TestWithProject: Setting up test project
2024-02-23T20:34:38.1923620Z .
2024-02-23T20:34:38.1924200Z The job test_a was saved and received the ID: 6
2024-02-23T20:34:38.1986260Z Loading and filtering jobs:   0%|          | 0/5 [00:00<?, ?it/s]
2024-02-23T20:34:38.2571030Z The job test_b was saved and received the ID: 7
2024-02-23T20:34:38.2572470Z Loading and filtering jobs: 100%|██████████| 5/5 [00:00<00:00, 714.39it/s]
2024-02-23T20:34:38.2573330Z 
2024-02-23T20:34:38.2573650Z The job test_c was saved and received the ID: 8
2024-02-23T20:34:38.2576000Z The job test_d was saved and received the ID: 9
2024-02-23T20:34:38.2577220Z The job test_table was saved and received the ID: 10
2024-02-23T20:37:30.7053850Z ##[error]The action has timed out.
2024-02-23T20:37:30.7381440Z Post job cleanup.
2024-02-23T20:37:30.9676170Z [command]/usr/local/bin/git version
2024-02-23T20:37:30.9815000Z git version 2.43.0
2024-02-23T20:37:31.1645960Z Copying '/Users/runner/.gitconfig' to '/Users/runner/work/_temp/abc94698-4efd-4a12-9c86-fc8991bb9d2b/.gitconfig'
2024-02-23T20:37:31.2653050Z Temporarily overriding HOME='/Users/runner/work/_temp/abc94698-4efd-4a12-9c86-fc8991bb9d2b' before making global git config changes
2024-02-23T20:37:31.3549000Z Adding repository directory to the temporary git global config as a safe directory
2024-02-23T20:37:31.3630790Z [command]/usr/local/bin/git config --global --add safe.directory /Users/runner/work/pyiron_base/pyiron_base
2024-02-23T20:37:31.3787240Z [command]/usr/local/bin/git config --local --name-only --get-regexp core\.sshCommand
2024-02-23T20:37:31.4287210Z [command]/usr/local/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :"
2024-02-23T20:37:31.5352650Z [command]/usr/local/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2024-02-23T20:37:31.5358390Z http.https://github.com/.extraheader
2024-02-23T20:37:31.5375810Z [command]/usr/local/bin/git config --local --unset-all http.https://github.com/.extraheader
2024-02-23T20:37:31.5388130Z [command]/usr/local/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :"
2024-02-23T20:37:31.6138710Z Cleaning up orphan processes
2024-02-23T20:37:32.6681650Z Terminate orphan process: pid (8769) (python3.12)
2024-02-23T20:37:32.7164470Z Terminate orphan process: pid (8766) (python3.12)
2024-02-23T20:37:32.7591070Z Terminate orphan process: pid (6738) (python3.12)
2024-02-23T20:37:32.7788160Z Terminate orphan process: pid (6697) (python3.12)
```

--- a/pyiron_base/jobs/job/generic.py
+++ b/pyiron_base/jobs/job/generic.py
@@ -1538,7 +1538,7 @@ class GenericJob(JobCore, HasDict):
         elif self._executor_type == "pympipool.mpi.executor.PyMPIExecutor" and platform.system() == "Darwin":
             # The Mac firewall might prevent connections based on the network address - especially Github CI
             return import_class(self._executor_type)(max_workers=max_workers, hostname_localhost=True)
-        elif isinstance(self._executor_type, str) and platform.system() == "Linux":
+        elif isinstance(self._executor_type, str):
             return import_class(self._executor_type)(max_workers=max_workers)
         else:
             raise TypeError("The self.executor_type has to be a string.")

--- a/pyiron_base/jobs/job/generic.py
+++ b/pyiron_base/jobs/job/generic.py
@@ -1535,9 +1535,14 @@ class GenericJob(JobCore, HasDict):
             raise ValueError(
                 "No executor type defined - Please set self.executor_type."
             )
-        elif self._executor_type == "pympipool.mpi.executor.PyMPIExecutor" and platform.system() == "Darwin":
+        elif (
+            self._executor_type == "pympipool.mpi.executor.PyMPIExecutor"
+            and platform.system() == "Darwin"
+        ):
             # The Mac firewall might prevent connections based on the network address - especially Github CI
-            return import_class(self._executor_type)(max_workers=max_workers, hostname_localhost=True)
+            return import_class(self._executor_type)(
+                max_workers=max_workers, hostname_localhost=True
+            )
         elif isinstance(self._executor_type, str) and platform.system() == "Linux":
             return import_class(self._executor_type)(max_workers=max_workers)
         else:

--- a/pyiron_base/jobs/job/generic.py
+++ b/pyiron_base/jobs/job/generic.py
@@ -9,6 +9,7 @@ from concurrent.futures import Future, Executor
 from datetime import datetime
 from inspect import isclass
 import os
+import platform
 import posixpath
 import warnings
 
@@ -1534,7 +1535,10 @@ class GenericJob(JobCore, HasDict):
             raise ValueError(
                 "No executor type defined - Please set self.executor_type."
             )
-        elif isinstance(self._executor_type, str):
+        elif self._executor_type == "pympipool.mpi.executor.PyMPIExecutor" and platform.system() == "Darwin":
+            # The Mac firewall might prevent connections based on the network address - especially Github CI
+            return import_class(self._executor_type)(max_workers=max_workers, hostname_localhost=True)
+        elif isinstance(self._executor_type, str) and platform.system() == "Linux":
             return import_class(self._executor_type)(max_workers=max_workers)
         else:
             raise TypeError("The self.executor_type has to be a string.")

